core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2772,7 +2772,7 @@ private void mountInternal(Supplier<JournalContext> journalContext, LockedInodeP
             NetworkAddressUtils.getConnectHost(NetworkAddressUtils.ServiceType.MASTER_RPC,
                 ServerConfiguration.global()));
         // Check that the ufsPath exists and is a directory
-        if (!ufs.isDirectory(ufsPath.toString())) {
+        if (!ufs.isExistingDirectory(ufsPath.toString())) {
```

Comment:
we don't know that it's existing at this point, so we should use regular `isDirectory`

Reply:
Changed, thanks

Reply:
agree, I think we can remove `isExistingDirectory`